deploy/scripts/deploy.ps1

```diff
@@ -68,13 +59,50 @@ while (!($secretName -match '^[a-zA-Z0-9,-,_]*$')) {
 }
 kubectl create secret docker-registry $secretName --docker-server $acrResources[1] --docker-username $servicePrincipalId --docker-password $servicePrincipalPassword --docker-email $dockerEmail
 
+Write-Output "Creating postgresql database server"
```

Comment:
Hva er det vi trenger postgresql for?

Reply:
Database for Gitea 🔢

Reply:
Essentially the same, both are azure maintained servers.

https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal

https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-portal

Or did you think of another Azure Resource?